datamodules/lucas.py

```diff
@@ -134,14 +139,13 @@ def setup(self, stage=None):
             dataset=data,
             case_number=self.case_number,
             new_machine=self.new_machine,
-            seed=self.seed,
         )
 
         if self.debug:
             train_inds = train_inds[:80]
             test_inds = test_inds[:20]
 
-        n_val = int(round(len(train_inds) * self.val_percent))
+        n_val = int(round(len(train_inds) * self.val_percent)) ## maybe cut these off the test set instead? 
```

Comment:
Nope, val always comes from the train set. Test set is basically the data that doesn't exist yet.

Reply:
correct, and done